apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -249,12 +249,21 @@ function load_block_patterns_from_api( $current_screen ) {
 		return;
 	}
 
-	if ( ! $current_screen->is_block_editor ) {
+	$is_site_editor = ( function_exists( 'gutenberg_is_edit_site_page' ) && gutenberg_is_edit_site_page( $current_screen->id ) );
+
+	if ( ! $current_screen->is_block_editor && ! $is_site_editor ) {
 		return;
 	}
 
+	$pattern_sources = array( 'block_patterns' );
```

Comment:
This is my typo, but I just noticed that this variable uses the singular `$pattern_sources` instead of the double plural `$patterns_sources` that we're using in the class. Shall we make them consistent? (Double plural?)

Reply:
Yes we should. My oversight on that one. Thanks for noticing.